### DISCUSSION TITLE
Problem with Windows Apps in Menu Start

### DISCUSSION THREAD

**Discussion by Patryk-Ordon** [valinet/ExplorerPatcher#1437] [Q&A]
![image](https://github.com/valinet/ExplorerPatcher/assets/82160814/1caf87d1-0ec4-4f7e-8c71-b6b2e77b4307)
All Windows UWP apps like Messenger are missing from Menu Start and All Apps, there are only x86 apps.

**Comment by pyrates999** [comment]
I think this is a duplicate of this: https://github.com/valinet/ExplorerPatcher/issues/1108
